test/js/components/ReadableDate-spec.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { assert } from 'chai';
+import { shallow } from 'enzyme';
+
+import { ReadableDate } from '../../../src/js/components';
```

Comment:
Yeah that sounds like a good approach to me. I know there's some more tweaks to come in testing, so I'll try to work that in soon.